.vscode/settings.json

```diff
@@ -12,5 +12,48 @@
     ".project": true
   },
   "C_Cpp.default.configurationProvider": "vscode-wpilib",
-  "wpilib.online": true

```

Comment:
Should `settings.json` be checked in? How did all this information end up in this file?